adapter-base-quarkus/src/test/java/org/eclipse/hono/adapter/quarkus/KafkaRuntimeConfigProducerTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.hono.adapter.quarkus;
+
+import javax.inject.Inject;
+
+import org.eclipse.hono.client.kafka.KafkaAdminClientConfigProperties;
+import org.eclipse.hono.client.kafka.KafkaProducerConfigProperties;
+import org.eclipse.hono.client.kafka.consumer.KafkaConsumerConfigProperties;
+import org.junit.jupiter.api.Assertions;
```

Comment:
can we please use `import static org.assertj.core.api.Assertions.assertThat` instead?

Reply:
I've been 10  minutes too late. Please see #2560.